blog/2024-04-26-license-questions.mdx

```diff
@@ -0,0 +1,68 @@
+---
+slug: license-questions
+title: License Questions
+authors: [parker]
+tags: [panel, license]
+hide_table_of_contents: true
+---
+
+As some of you know, people have been asking us questions about when you need a license or not.
+
+I asked for people to get me questions they wanted clarifications on and here are some answers. 
+
+## modification questions
+
+1.  Q: What are the plans for the theming part of Pelican Panel and what will the front end look like?
```

Comment:
```suggestion
1.  Q: What are the plans for the theming Pelican Panel and what will it look like?
```

I want to stay away from "front end" because it is ambiguous. For example, I use it to refer to everything that is not /admin (/panel). But some people refer to React.js as the front end and some refer to all HTML/CSS as the front end.

Reply:
This is the question as asked. I am not editing other peoples words.